src/lambda/explorer/cloudFormationNodes.ts

```diff
@@ -68,7 +73,7 @@ export class DefaultCloudFormationNode extends AWSTreeErrorHandlerNode implement
             this.stackNodes,
             stacks.keys(),
             key => this.stackNodes.get(key)!.update(stacks.get(key)!),
-            key => new DefaultCloudFormationStackNode(this, stacks.get(key)!)
+            key => new DefaultCloudFormationStackNode(this, stacks.get(key)!, this.getExtensionAbsolutePath)
```

Comment:
Be careful about passing unbound instance methods around. Either wrap it in a lambda:

```typescript
(arg1, arg2, /* etc  */) => this.getExtensionAbsolutePath(arg1, arg2, /* etc */)
```

or bind it:

```typescript
this.getExtensionAbsolutePath.bind(this)
```